app/src/layouts/adminLayout/header/header.jsx

```diff
@@ -39,29 +44,23 @@ adminHeaderCrumb, onClickMenu, isMenuOpen }) => (
     </div>
     <div className={cx('container')}>
       <h3 className={cx('header-name')}>
-        <FormattedMessage
-          id={'AdminHeader.header'}
-          defaultMessage={'Management board'}
-        />
+        <FormattedMessage id={'AdminHeader.header'} defaultMessage={'Management board'} />
         <span className={cx('header-crumb')}>{adminHeaderCrumb}</span>
       </h3>
       <div className={cx('admin-header-controls')}>
-        <button className={cx({ 'back-to-project': true, btn: true })} onClick={onClickBackToProject}>
-          <FormattedMessage
-            id={'AdminHeader.btnToProject'}
-            defaultMessage={'Back to project'}
-          />
+        <button
+          className={cx({ 'back-to-project': true, btn: true })}
```

Comment:
`className={cx('back-to-project', 'btn')}`

Reply:
Fixed